pkgs/by-name/az/azure-functions-core-tools/package.nix

```diff
@@ -1,17 +1,19 @@
-{ lib,
+{
+  lib,
   stdenv,
   fetchFromGitHub,
   buildDotnetModule,
   buildGoModule,
   dotnetCorePackages,
+  versionCheckHook,
```

Comment:
Unused, but its not worth blocking on this. Unfortunate that upstream doesn't properly update their version
```suggestion
```